virttest/utils_windows/system.py

```diff
@@ -44,3 +44,18 @@ def os_arch(session):
     :return: Windows OS architecture.
     """
     return _osinfo(session, props=["OSArchitecture"])
+
+
+def file_existing(session, filename):
+    """
+    Check if a file is existing.
+
+    :param session: Session object.
+    :param filename: File name with full path.
+    :return: bool value: True if file exists.
+    """
+    check_cmd = "dir %s" % filename
```

Comment:
To handle hidden files and systems files the command should be appended with option `/a`
```
C:\>dir c:\Users\Administrator\Desktop\desktop.ini
 Volume in drive C is Windows
 Volume Serial Number is 8C27-4256

 Directory of c:\Users\Administrator\Desktop

File Not Found

```
```
C:\>dir /a c:\Users\Administrator\Desktop\desktop.ini
 Volume in drive C is Windows
 Volume Serial Number is 8C27-4256

 Directory of c:\Users\Administrator\Desktop

07/27/2018  03:42 AM               282 desktop.ini
               1 File(s)            282 bytes
               0 Dir(s)  16,444,526,592 bytes free

```
Also recommend adding option `/b` to reduce log size.